sale_merge_draft_invoice/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License LGPL-3
+
+========================
+Sale Merge Draft Invoice
+========================
+
+This module merges new invoices with existing draft ones when creating them.
+
+The merge between invoices is done when the partner and the currency are the
+same.
+
```

Comment:
I think this section should reference #568 capabilities and Odoo default capabilities.  The way it reads now, it reads like this module groups by partner and currency, when in fact it doesn't, it just considers draft invoices in the grouping. 

Explain the default group of Odoo like you have, then explain that alternative custom groupings can be achieved using that module.

Something like.

Odoo can merge sales orders into a single invoice, grouped by partner and currency, when the sales orders are selected and invoiced together.  This module extends the grouping to also consider any pre-existing draft invoices and merge them.  

If you also need to group on fields other than partner and currency, the module sale_invoice_group_method allows you to specify alternative grouping methods and is fully compatible with this module.